sklearn/linear_model/tests/test_quantile.py

```diff
@@ -28,7 +28,7 @@ def X_y_data():
 
 @pytest.fixture
 def default_solver():
```

Comment:
You can remove this function and adapt the test that needs it by not specifying `solver=...` in them

Reply:
Removed and adapted the tests.